src/pytkdocs/objects.py

```diff
@@ -22,9 +22,8 @@
 from pathlib import Path
 from typing import List, Optional, Union
 
-from pytkdocs.parsers.docstrings import parse
-
-from .properties import NAME_CLASS_PRIVATE, NAME_PRIVATE, NAME_SPECIAL, ApplicableNameProperty
+from pytkdocs.parsers.docstrings.google import Google as DocstringParser
```

Comment:
What will be the expected way to change which parser is being used? Is this going to be a global configuration that gets set or a local configuration that gets passed to the `Loader`?

Reply:
I have not thought about that yet, I'm open to suggestions :slightly_smiling_face: 

But passing the style to use as an argument to the `Loader` seems like a good idea!

Reply:
Ok, then it might be good to define a dictionary mapping parser name to the class definition in `pytkdocs.parsers.docstrings.__init__`
Something like:
``` python
PARSERS = {"Google": Google}
```

This can be imported in Loader and then a `docstring_style` parameter would make it so the `Loader` can choose which parser to use

Reply:
I'm afraid importing parser implementations in the `__init__` module would create cyclic imports, but yes, the mapping is the way to go :slightly_smiling_face:

Reply:
Oh, good point. Either of your suggestions should work.

Reply:
I moved them into a `base` module.